### PR TITLE
Basic lift methods

### PR DESCRIPTION
**Pull request by classygirl44**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/instances/Lift.java [hunk 1]
```diff
@@ -7,24 +7,119 @@
...
+    rightMotorLeader.setNeutralMode(NeutralMode.Brake);
+    leftMotorFollower.setNeutralMode(NeutralMode.Brake);
+    rightMotorFollower.setNeutralMode(NeutralMode.Brake);
+
+    leftMotorLeader.configPeakOutputForward(1.0);
+    rightMotorLeader.configPeakOutputReverse(-1.0);
+    leftMotorFollower.configPeakOutputForward(1.0);
+    rightMotorFollower.configPeakOutputReverse(-1.0);
+
+    //leftMotorLeader.setInverted(true);
```
Comment (Silvery-Sheen) [inline src/main/java/frc/robot/subsystems/instances/Lift.java:55]: Whadisdis

Comment (classygirl44) [reply src/main/java/frc/robot/subsystems/instances/Lift.java]: idc

Comment (Silvery-Sheen) [reply src/main/java/frc/robot/subsystems/instances/Lift.java:55]: Pleas4 remove

Comment (classygirl44) [reply src/main/java/frc/robot/subsystems/instances/Lift.java:55]: We weren't sure if we were going to need it so we commented it out just in case.

**Review by Silvery-Sheen — COMMENTED** [review]
Shouldn't you define all of the objects so that the robot doesn't break when they are getting called on

File: src/main/java/frc/robot/subsystems/instances/Lift.java [hunk 2]
```diff
@@ -7,24 +7,119 @@
...
 
 public class Lift extends SubsystemBase implements LiftInterface {
   /**
    * Creates a new Lift.
    */
+  private static WPI_TalonSRX leftMotorLeader;
+  private static WPI_TalonSRX rightMotorLeader;
+  private static WPI_TalonSRX leftMotorFollower;
+  private static WPI_TalonSRX rightMotorFollower;
+  private static Compressor compressor;
```
Comment (jjesse1972) [inline src/main/java/frc/robot/subsystems/instances/Lift.java]: I dont think we want to control the compressor from this subsystem. We should probably remove.

File: src/main/java/frc/robot/subsystems/instances/Lift.java [hunk 3]
```diff
@@ -7,24 +7,119 @@
...
+    rightMotorLeader.configPeakOutputReverse(-1.0);
+    leftMotorFollower.configPeakOutputForward(1.0);
+    rightMotorFollower.configPeakOutputReverse(-1.0);
+
+    //leftMotorLeader.setInverted(true);
+
+    leftMotorFollower.follow(leftMotorLeader);
+    rightMotorFollower.follow(rightMotorLeader);
 
+    compressor = new Compressor(16);
```
Comment (jjesse1972) [inline src/main/java/frc/robot/subsystems/instances/Lift.java]: remove compressor

**Review by jjesse1972 — CHANGES_REQUESTED** [review]

**Review by kcjesse02 — APPROVED** [review]

**Review by jjesse1972 — APPROVED** [review]

**Pull request merged**